app/components/blocks/AboutContent/aboutContent.tsx

```
@@ -1,9 +1,10 @@
 import React from "react";
-import type { AboutBlocksAboutContent } from "@tina/__generated__/types";
+
+import HowWeWork from "./sections/HowWeWork";
 import Introduction from "./sections/Introduction";
```

ESLint

[error] 'Introduction' is defined but never used.

(@typescript-eslint/no-unused-vars)